src/OrchardCore.Modules/OrchardCore.Tenants/Startup.cs

```diff
@@ -153,6 +154,7 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddScoped<FeatureProfilesManager>();
             services.AddScoped<IFeatureProfilesService, FeatureProfilesService>();
             services.AddScoped<IFeatureProfilesSchemaService, FeatureProfilesSchemaService>();
+            services.AddScoped<TenantValidator>();
```

Comment:
Why is it needed separately for FeatureProfiles too?

Reply:
No need